src/centralserver/admin-service/src/main/java/org/niis/xroad/centralserver/restapi/converter/ClientConverter.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * The MIT License
+ * Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.centralserver.restapi.converter;
+
+import lombok.RequiredArgsConstructor;
+import org.niis.xroad.centralserver.openapi.model.Client;
+import org.niis.xroad.centralserver.openapi.model.ClientId;
+import org.niis.xroad.centralserver.restapi.dto.FlattenedSecurityServerClientDto;
+import org.springframework.stereotype.Component;
+
+import java.time.ZoneOffset;
+
+@Component
+@RequiredArgsConstructor
+public class ClientConverter {
```

Comment:
Converters could be changed to not be Spring beans

Reply:
Addressed in https://github.com/nordic-institute/X-Road/commit/2cab7b89cd5ce33e55c88429cca81785f6e0b10a

Convention documented in confluence.